src/keys.rs

```diff
@@ -112,7 +116,7 @@ impl From<&SpendingKey> for SpendAuthorizingKey {
 /// $\mathsf{ak}$ but stored here as a RedPallas verification key.
 ///
 /// [orchardkeycomponents]: https://zips.z.cash/protocol/nu5.pdf#orchardkeycomponents
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialOrd, Ord)]
 pub struct SpendValidatingKey(redpallas::VerificationKey<SpendAuth>);
```

Comment:
Since we're directly importing `VerificationKey` now.
```suggestion
pub struct SpendValidatingKey(VerificationKey<SpendAuth>);
```

Reply:
I would prefer that we _didn't_ directly import `VerificationKey`, and instead consistently use `redpallas::VerificationKey` for clarity.